### PR TITLE
 Added code of daily Coding Exercise-17

### PR DESCRIPTION
**Pull request by anushreeSonak**

### CODE REVIEW

File: First/dailyExercises/InsertString.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
+package firstPackage.First.dailyExercises;
+
+public class InsertString {
+    // Function to insert string
+    public static String insertString(String originalString, String stringToBeInserted, int var) {
```
Comment (AmrutaDhage20) [inline First/dailyExercises/InsertString.java]: Please give proper variable name for var.

Comment (AmrutaDhage20) [inline First/dailyExercises/InsertString.java]: Please give proper variable name for var.

File: First/dailyExercises/InsertString.java [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+        // Create a new string
+        String newString = originalString.substring(0, var + 1) + stringToBeInserted + originalString.substring(var + 1);
+        return newString;
+    }
+
+    public static void main(String[] args) {
+        // Get the Strings
+        String originalString = "Hello  World ";
+        String stringToBeInserted = "IT";
```
Comment (AmrutaDhage20) [inline First/dailyExercises/InsertString.java:14]: give proper string name

File: First/dailyExercises/InsertString.java (hunk 1 above)
Comment (anushreeSonak) [reply First/dailyExercises/InsertString.java]: DONE !

Comment (anushreeSonak) [reply First/dailyExercises/InsertString.java]: DONE !

**Review by AmrutaDhage20 — APPROVED** [review]

**Review by amitsahu8 — APPROVED** [review]

**Pull request merged**